doc/nrf/ug_zigbee_commissioning.rst

```diff
@@ -0,0 +1,214 @@
+
+  Target: This node identifies itself, and receives and responds to requests from
+  the initiator
+
+Network Steering
+================
+
+The Network Steering mode is the first step towards commissioning a device.
+
+.. figure:: /images/zigbee_commissioning_steering.svg
+   :alt: Zigbee Network Steering procedure
+
+   Zigbee Network Steering procedure
+
+How the node acts in this mode depends on whether the node is in a Zigbee network or not:
+
+* If the node is already member of a Zigbee network, it opens the network and then closes it after the specified period of time (180 seconds by default).
+  The opening of the network is triggered when any of the nodes broadcasts the permit join request.
+  Broadcasting the request can also be started manually, for example when you press the associated button on the :ref:`Zigbee Coordinator node <zigbee_network_coordinator_sample>`.
+* If the node is not a member of a Zigbee network, it scans channels for an open network to associate with.
+  Once it finds a suitable network, it attempts to join it and get the network key.
+  If the node finds more than one available network, it will attempt to join them one by one.
+
+If the node does not find a network to join and it has either the Coordinator or the Router role, it can attempt to form a new network.
+If the node has the End Device role, its commissioning stops here.
+
+The new device must be authenticated and authorized to become part of the network.
+For `Centralized security network`_, the authentication and authorization also involves establishing a unique Trust Center Link Key (TCLK).
+
+Network Formation
+=================
+
+The Network Formation mode is available only to Coordinators and Routers, because End Devices are not capable of forming a Zigbee network.
+
+.. figure:: /images/zigbee_commissioning_formation.svg
+   :alt: Zigbee Network Formation procedure
+
+   Zigbee Network Formation procedure
+
+In this mode, the node has not found a network to join during the Network Steering mode, and so it attempts to form a new network.
+Depending on the node role:
+
+* A Coordinator forms a centralised security network.
+* A Router forms a distributed security network.
+
+If Network Formation is successful, you can add nodes to the new network using `Network Steering`_.
+If unsuccessful, the commissioning procedure stops.
+
+Centralized security network
+----------------------------
+
+The centralized security network is started by the Zigbee Coordinator node, which assumes the Trust Center functionality.
+As such, it is responsible for managing the security and authorization in the network.
+In the centralized secutrity network, only the Trust Center can provide the network key to the joining device.
+This happens by using install code link keys specific to the joining device.
+The install code link key is exchanged with the Trust Center, which allows for encrypting packets exchanged between the Trust Center and the new node.
```

Comment:
The second clause of this sentence, should it be:
`which allows for encrypting packets to be exchanged ...` ?